Makefile.pre.in

```diff
@@ -3106,6 +3106,11 @@ config.status:	$(srcdir)/configure
 Python/asm_trampoline.o: $(srcdir)/Python/asm_trampoline.S
 	$(CC) -c $(PY_CORE_CFLAGS) -o $@ $<
 
+Python/emscripten_trampoline_inner.wasm: $(srcdir)/Python/emscripten_trampoline_inner.c
+	$$(dirname $$(dirname $(CC)))/bin/clang -o $@ $< -mgc -O2 -Wl,--no-entry -Wl,--import-table -Wl,--import-memory -target wasm32-unknown-unknown -nostdlib
```

Comment:
Not sure I follow what the "dirname of dirname" is doing here - doesn't this assume that `emcc` (which should be the value of CC) is installed in /usr/bin?

Reply:
I added a comment explaining.